python/cudf/cudf/utils/applyutils.py

```diff
@@ -143,17 +144,20 @@ def run(self, df, **launch_params):
         # Get input columns
```

Comment:
Need to acquire a spill lock for this function.

Reply:
Done